src/js/apps/patients/patient/flow/flow_app.js

```diff
@@ -98,7 +97,24 @@ export default SubRouterApp.extend({
     this.stopChildApp('actionSidebar');
   },
   subscribe() {
-    Radio.request('ws', 'subscribe', [this.flow, ...this.actions.models]);
+    const channel = Radio.channel('ws');
+    const filters = { actions: { flow: this.flow.id } };
+
+    channel.request('subscribe', [this.flow, ...this.actions.models], { filters });
+
+    this.listenTo(channel, 'message', ({ category, resource }) => {
+      if (category !== 'ResourceCreated') return;
```

Comment:
I think we should be safe just doing this check. On the flow page, we only need to handle a `ResourceCreated` category message. Which would only happen for an action, not a flow.